src/App.jsx

```diff
@@ -27,6 +28,7 @@ function App() {
         <Route path="/donners" element={<Donners />} />
         <Route path="/causes/:id" element={<CauseDetails />} />
         <Route path="/donate" element={<DonatePage />} />
+        <Route path="/careerdetails" element={<CareerDetails />} />
```

Comment:
Check cause details route definition on line 29.
CareerDetails should have a /:id to it